andela/travel_tool_front
# #160751466 Show toaster when notification is sent

#### What does this PR do?
Displays a toast when a user receives a notiication

#### Description of Task to be completed?
- add toast message when notification is sent

#### How should this be manually tested?
- Clone the repo
- Change into the project directory
- Check out this branch with `git checkout ch-show-notification-toast-160751466`
- Start the server `make start`
- Setup the backend by following this [README.md](https://github.com/andela/travel_tool_back)
- In a browser, navigate to `localhost:3000`
- Perform an action that requires you to get a notification(create a request, approve a request)
- You should see a notification

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160751466](https://www.pivotaltracker.com/story/show/160751466)

#### Screenshots (if appropriate)
<img width="1426" alt="screen shot 2018-09-26 at 12 26 23 pm" src="https://user-images.githubusercontent.com/25783086/46077058-6e4bb080-c187-11e8-8e11-6b99c0d62fe4.png">


#### Questions:
None
